docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -138,11 +138,11 @@ Step 2: Generate a User Key
 Generate your user key for Sawtooth, using the same terminal window as the
 previous step.
 
-   .. code-block:: console
+.. code-block:: console
 
-      user@validator$ sawtooth keygen
-      writing file: /home/yourname/.sawtooth/keys/yourname.priv
-      writing file: /home/yourname/.sawtooth/keys/yourname.pub
+   user@validator$ sawtooth keygen my_key
```

Comment:
Since this `my_key` is optional, should a note be added that the default key name would be `$USER`?

Reply:
Good idea. I'll add this info.

Reply:
Done - added a commit with the note (in two places).